test/2d/test_2d.cpp

```diff
@@ -121,6 +121,9 @@ TEST (Convolution, borderOptions)
     EXPECT_NEAR ((*output_cloud)(0,i).intensity, ((*input_cloud)(1,i).intensity), 1);
     EXPECT_NEAR ((*output_cloud)(width-1,i).intensity, (-(*input_cloud)(width-2,i).intensity), 1);
   }
+
+  delete(k);
+  delete(conv);
```

Comment:
The variables `k` and `conv` don't need to be pointer in the first place.

Reply:
Should I make these and the other pointers normal variables?

Reply:
Exactly. :+1: